sdk/appconfiguration/app-configuration/samples/react-sample/src/components/Page.tsx

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+/*
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT license.
+*/
+
+import React from "react";
+import { useEffect, useState } from "react";
+import { getEnvironmentVariable } from "../utils";
+import {
+  AppConfigurationClient,
+  featureFlagPrefix,
+  isFeatureFlag,
+  isFeatureFlagClientFilter
+} from "@azure/app-configuration";
+
+let client: AppConfigurationClient;
+
+const featureFlag1 = {
+  id: "react-app-feature-1",
+  description: "",
+  enabled: false,
+  conditions: {
+    client_filters: []
+  }
+};
+const featureFlag2 = {
+  id: "react-app-feature-2",
+  description: "",
+  enabled: true,
+  conditions: {
+    client_filters: [
+      {
+        name: "Microsoft.TimeWindow",
+        parameters: {
+          Start: "Mon, 12 Apr 2021 07:00:00 GMT",
+          End: "Wed, 14 Apr 2021 07:00:00 GMT"
+        }
+      }
+    ]
+  }
+};
+
+export default function Page(): JSX.Element {
+  const feature1Name = "react-app-feature-1";
+  const feature2Name = "react-app-feature-2";
+  const [feature1, setFeature1] = useState<{ enabled: boolean }>({ enabled: false });
+  const [feature2, setFeature2] = useState<{ enabled: boolean }>({ enabled: false });
+
+  const getFeatureFlags = async (keys: string[]): Promise<void> => {
+    const [setting1, setting2] = await Promise.all(
+      keys.map((key: string) =>
+        client.getConfigurationSetting({
+          key: featureFlagPrefix + key
+        })
+      )
+    );
+    if (isFeatureFlag(setting1)) {
+      // console.log(`${setting1.key} is enabled : ${setting1.enabled}`, setting1);
+      setFeature1({ enabled: setting1.enabled });
+    }
+    if (isFeatureFlag(setting2)) {
+      // console.log(`${setting2.key} is enabled : ${setting2.enabled}`, setting2);
+      const clientFilter = setting2.conditions.clientFilters?.[0];
+      if (isFeatureFlagClientFilter("timeWindow", clientFilter)) {
+        const now = Date.now();
+        const withinRange =
+          now - Date.parse(clientFilter.parameters.start) > 0 &&
+          Date.parse(clientFilter.parameters.end) - now > 0;
+        setFeature2({ enabled: withinRange });
+      }
+    }
+  };
+
+  useEffect(() => {
+    const connectionString = getEnvironmentVariable("REACT_APP_APPCONFIG_CONNECTION_STRING");
+    client = new AppConfigurationClient(connectionString);
+    getFeatureFlags([feature1Name, feature2Name]);
+  }, []);
+
+  return (
+    <React.Fragment>
+      <nav className="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
I've picked this snippet here https://getbootstrap.com/docs/4.0/components/navbar/#supported-content.

Are you suggesting using react-bootstrap instead? 
(If so, @maorleger suggested it might be a simpler sample with regular bootstrap. https://github.com/Azure/azure-sdk-for-js/pull/14845#discussion_r612426314)

Reply:
I agree with Will about mixing jQuery based bootstrap and React, but I think I was advocating going the other way here - grab the bootstrap CSS from CDN, use the grid system and buttons, etc. but keeping the sample simple without the JS parts. 

Adding react-bootstrap into the mix or adding custom nav components really doesn't help a new developer understand how to use the Azure SDK in React IMO and adds more concepts to what should be a simple sample focused around interacting with the Azure SDKs. 

If we can simplify the UI here and remove the jQuery and JS assets, just using CSS and common HTML elements I think it would make for a simpler example that is easier to understand for a new developer.

Reply:
Hmmm.. I'll look at simplification.

Reply:
Removed remove jquery and bootstrap js and converted the dropdown example to a button as it requires jquery.
https://github.com/Azure/azure-sdk-for-js/pull/14845/commits/a7942a037c3529e36ae82d9f92ff0b6e4114a1ba